### PR TITLE
ALS-93 Permit aMQ to send cloudwatch logs with new Resource Based IAM…

### PR DESCRIPTION
**Pull request by MartinYSpasov**
… policy

### CODE REVIEW

File: amazonmq/main.tf [hunk 1]
```diff
@@ -31,6 +31,8 @@ locals {
 
   # Construct the credentials_ssm_path from the common remote state variables tags.application and tags.environment_name and the fixed string
   credentials_ssm_path = "/${data.terraform_remote_state.vpc.environment_name}/${data.terraform_remote_state.vpc.tags.application}/weblogic/spg-domain"
+
+  env_prefix  = "${var.SPG_ENVIRONMENT_CODE}"
```
Comment (crimesp) [inline amazonmq/main.tf]: Please use `hmpps_asset_name_prefix` pattern as it enables us to be decoupled from the spg app and means all assets are in line and get the same prefix - this should work...

   env_prefix   = "${local.hmpps_asset_name_prefix}"

File: amazonmq/variables.tf [hunk 2]
```diff
@@ -1,3 +1,5 @@
+variable SPG_ENVIRONMENT_CODE {}
```
Comment (crimesp) [inline amazonmq/variables.tf]: possibly not needed with change from above

File: amazonmq/main.tf (hunk 1 above)
Comment (MartinYSpasov) [reply amazonmq/main.tf]: done

File: amazonmq/variables.tf (hunk 2 above)
Comment (MartinYSpasov) [reply amazonmq/variables.tf]: done

**Pull request merged**